Pipfile

```diff
@@ -0,0 +1,29 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+logzero = ">=1.3.1"
+feedparser = ">5.1"
+pandas = ">=0.22.0"
+numpy = ">=1.14.0"
+gensim = ">=3.2.0"
+scikit-learn = ">=0.19.1"
+scipy = ">=1.1.0"
+nltk = ">=3.3"
+pytest = "*"
+tox = "*"
+coverage = "*"
+pytest-cov = "*"
+pytest-flask = "*"
+gunicorn = "*"
+"flake8" = "*"
+ujson = "*"
+flask = "*"
+flask-Script = "*"
```

Comment:
we should remove flask-script and use native flask (flask run) instead....

Reply:
Yes, I created a separate PR for that (#14 ) but right now this would break the start of the app :)